build_tools/python/benchmark_suites/iree/vulkan_nvidia_benchmarks.py

```diff
@@ -0,0 +1,86 @@
+## Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Defines IREE Vulkan NVIDIA benchmarks."""
+
+from typing import List, Tuple, Sequence
+from benchmark_suites.iree import module_execution_configs
+from e2e_test_framework import unique_ids
+from e2e_test_framework.definitions import common_definitions, iree_definitions
+from e2e_test_framework.device_specs import device_collections
+from e2e_test_framework.models import model_groups
+import benchmark_suites.iree.utils
+
+
+class Linux_Vulkan_NVIDIA_Benchmarks(object):
+  """Benchmarks on Linux Vulkan NVIDIA devices."""
+
+  AMPERE_TARGET = iree_definitions.CompileTarget(
+      target_architecture=common_definitions.DeviceArchitecture.NVIDIA_AMPERE,
+      target_backend=iree_definitions.TargetBackend.VULKAN_SPIRV,
+      target_abi=iree_definitions.TargetABI.VULKAN_LINUX)
+  PASCAL_TARGET = iree_definitions.CompileTarget(
+      target_architecture=common_definitions.DeviceArchitecture.NVIDIA_PASCAL,
+      target_backend=iree_definitions.TargetBackend.VULKAN_SPIRV,
+      target_abi=iree_definitions.TargetABI.VULKAN_LINUX)
+
+  COMPILE_EXTRA_FLAGS = [
+      "--iree-stream-resource-index-bits=64", "--iree-vm-target-index-bits=64",
+      "--iree-preprocessing-pass-pipeline=builtin.module(func.func("
+      "iree-flow-detach-elementwise-from-named-ops,"
+      "iree-preprocessing-convert-conv2d-to-img2col,"
+      "iree-flow-convert-1x1-filter-conv2d-to-matmul,"
+      "iree-preprocessing-pad-linalg-ops{pad-size=32}))"
+  ]
```

Comment:
Having implicit string concatenation across lines and list elements across lines is pretty confusing, IMO. Especially when the line elements have a comma inside the quote. Can we wrap this in parens, add an explicit `+` or somehow make this clearer

```suggestion
    COMPILE_EXTRA_FLAGS = [
        "--iree-stream-resource-index-bits=64",
        "--iree-vm-target-index-bits=64",
        ("--iree-preprocessing-pass-pipeline=builtin.module(func.func("
         "iree-flow-detach-elementwise-from-named-ops,"
         "iree-preprocessing-convert-conv2d-to-img2col,"
         "iree-flow-convert-1x1-filter-conv2d-to-matmul,"
         "iree-preprocessing-pad-linalg-ops{pad-size=32}))"),
    ]
```

Reply:
Good points. Changed to implementation to use two lists, which should be more readable now!